project/ws/app/src/lib/routes/app-toc/components/app-toc-content-card/app-toc-content-card.component.ts

```diff
@@ -90,6 +90,31 @@ export class AppTocContentCardComponent implements OnInit, OnChanges {
             })
           }
         }
+        let arr = []
+        data['result']['contentList'].forEach((dataResult:any) => {
+          if (dataResult.completionPercentage === 100) {
+            arr.push(dataResult)
+          }
+        })
+        if (arr.length === data['result']['contentList'].length) {
+          const obj = {
+            request: {
+              batchId: this.batchId,
+              courseId: this.contentId,
+              userIds: [userId]
+            }
+          }
+          this.contentSvc.processCertificate(obj).subscribe(
+            result => {
+              // tslint:disable-next-line:no-console
+              console.log(result)
```

Comment:
remove console.log

Reply:
Yes, I know. I had added it, because the content progress API was not working then and hence had added  the tslint  
// tslint:disable-next-line:no-console